components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -221,6 +221,14 @@ - (CGSize)sizeThatFits:(__unused CGSize)size {
   return CGSizeMake(edge, edge);
 }
 
+- (void)setHidden:(BOOL)hidden {
+  [super setHidden:hidden];
+
+  if (hidden) {
```

Comment:
Please test this PR against internal clients before submitting to make sure we don't break any screenshot tests.

Reply:
Tested internally and nothing is broken.